UX_PRINCIPLES.md

```diff
@@ -0,0 +1,79 @@
+- Don’t expose your ‘inside baseball’ to users. Re-word things for them and avoid jargon or overly complex technical terminology.
+- As features develop and grow over time, the concepts and terminology may begin to break down and lead to confusion. Don’t be afraid to change a command later if it no longer makes sense.
+### Share Context
+- Map out the commands/subcommands/flags in a visual diagram
+- Walk the team through the diagram and post it in an easily accessible place
+- Refer to the map, as a team, when new c/s/f are going to be added (i.e. as new features are developed)
+### Be a Good Host
+- Provide guidepost commands that help people get started quickly and avoid common pitfalls
+ - For example, use a ‘getting started’ command to streamline rote one-time setup and decrease the time it takes for a user to get to the ‘good stuff’ (i.e. shorten the time to delight)
+- If you find something unnecessarily (or annoyingly) complex as you use the product yourself, then it’s probably generating the same negative response for others. Consider how to simplify the flow and remove the potholes.
+- Familiarity, simplicity and readability make it memorable. Docs are useful, but we don’t want people having to continually refer back to them.
+### Explain Yourself
+- Comment your code, be transparent, and use these as notes when building your help docs. If you have a public repo, then you can direct users there later to better understand the system and contribute (in addition to docs).
+
+## Output
+### Offer Timely Help
+- Provide targeted/contextual help - make assumptions based upon their current setup (have they completed the install step?)
+- Use flags to scope the help docs (normal/frequent, all, install, emergency)
+- In times of emergency, provide a clear path to a human at Chef support (if possible) - remind of chef support email or phone number within help docs
+### Embrace Design Constraints
+- Use characters (icons), progress bars, and color in your output
+- Provide an option to remove characters, progress bars and color (can break automated build tasks, for example, that humans aren’t reading/watching anyway)
+### Generate Humane Output
+- Make output human readable and concise by default. Offer more/less verbose output as an option via a flag.
+- Examine the current state of the system and make output contextual
+- Identify for analogs and antilogs. Do repeat CLI experiences that you’ve enjoyed elsewhere; Don’t repeat mistakes others have made. (e.g. In Habitat, building a package follows a format like bundle install)
+- Write output with a clear messages that a) sets the stage with “what we’re about to do”, b) progress “what is happening now”, and c) “did it fail or succeed?” and if it failed explain why, then provide a recommendation/next step. If it succeeds, celebrate it with memorable/playful message.
```

Comment:
Based on the rest of the sentence, I'm guessing "messages" should be singular here.